Engine/Engine.cs

```diff
@@ -154,7 +154,7 @@ private static RunStepResult BuildSolutionSnapshot(IRunExecutorHost host, RunSta
                     Environment.GetEnvironmentVariable("ProgramFiles(x86)"),
                     string.Format(@"MSBuild\{0}\Bin\msbuild.exe", host.HostVersion)),
                 string.Format(
-                    @"/m /v:minimal /p:Configuration=Debug /p:CreateVsixContainer=false /p:DeployExtension=false /p:CopyVsixExtensionFiles=false /p:OutDir={1} {2}",
+                    @"/m /v:minimal /p:Configuration=Debug /p:CreateVsixContainer=false /p:DeployExtension=false /p:CopyVsixExtensionFiles=false /p:OutDir=""{1}\\"" ""{2}""",

```

Comment:
I found it didn't work for me without the trailing slash. I've seen comments on the internet that suggest that it's required, but I can't seem to find anything on MSDN at the moment.

Certainly it is intentional that I have it in the PR.

There is also some suggestion in internet comments that `OutputPath` may be used instead of `OutDir`.

https://msdn.microsoft.com/en-us/library/bb629394.aspx